src/components/home/Hero/style.module.scss

```diff
@@ -0,0 +1,148 @@
+@use 'src/styles/vars.scss' as vars;
+
+.inline {
+  display: inline;
+}
+
+@media screen and (width <= vars.$breakpoint-md) {
+  .subheading {
+    font-size: 1rem !important;
```

Comment:
I think it's one of the downsides of the `Typography` component, that it's hard to define multiple sizes for different screen sizes. Perhaps ideally, `Typography` should use globally defined classes instead of setting `style`, so it can be more easily overridden, and the CSS file can use media queries to define different font sizes for different screen sizes